scout/server/links.py

```diff
@@ -819,6 +819,15 @@ def alamut_gene_link(
         url_template(str): link to Alamut browser
     """
 
+    def get_gene_canonical_tx(gene: dict) -> str:
```

Comment:
This is good and should be solving the issue! Do you remember or have you found out why the issue occurs? We seem to have this kind of fallback construction in several places, either with dual checking on gene and transcript or simply falling back to checking the transcripts directly. Is it historical, or are we missing something in build?